client/src/context/auth/AuthState.js

```diff
@@ -75,12 +75,13 @@ const AuthState = (props) => {
 
   // Get Logged In User
   const getLoggedInUser = async () => {
-    console.log(axios.defaults.headers)
     if (localStorage.token) {
       setAuthToken(localStorage.token)
     }
     try {
-      const response = await axios.get('http://localhost:5000/api/user/loggedInUser')
+      const response = await axios.get(
+        'http://localhost:5000/api/user/loggedInUser'
```

Comment:
consider using a variable for your api url for ease of access to change for different envs